docs/src/administration/cli/api-tokens.md

```diff
@@ -153,19 +167,24 @@ To run a cron only on your production environment, check the environment type as
 
 ```yaml
 crons:
-    backup:
-        spec: 'H 4 * * *'
+    sendemails:
+        spec: '*/7 * * * *'
         commands:
             start: |
                 if [ "$PLATFORM_ENVIRONMENT_TYPE" = production ]; then
-                   platform backup:create --yes --no-wait
+                   cd public && send-pending-emails.sh
```

Comment:
:lady_beetle: This command might might sense in general, but it doesn't really work on a page about using the CLI. It should be a CLI command. I think your previous example was a good one. :shrug: